src/main/java/ua/lviv/iot/parkingServer/logic/ParkingService.kt

```diff
@@ -0,0 +1,46 @@
+package ua.lviv.iot.parkingServer.logic
+
+import org.springframework.stereotype.Service
+import ua.lviv.iot.parkingServer.datastorage.ParkingFileStore
+import ua.lviv.iot.parkingServer.model.Parking
+import java.util.concurrent.atomic.AtomicLong
+import javax.annotation.PostConstruct
+import javax.annotation.PreDestroy
+import ua.lviv.iot.parkingServer.logic.exception.EntityNotFoundException
+
+@Service
+class ParkingService(private val parkingFileStore: ParkingFileStore) {
+    private val parking: MutableMap<Long, Parking> = HashMap()
+    private var id: AtomicLong = AtomicLong(1L)
+
+    fun findAllParking(): List<Parking> = ArrayList<Parking>(parking.values)
```

Comment:
Kotlin compiler can easily infer ArrayList type since you declared `Parking` type in the return type: `List<Parking>`

so your code could be a bit shorter:
```kotlin
fun findAllParking(): List<Parking> = ArrayList(parking.values)
```

Don't ignore the IDEA tips :)

Reply:
The next step:
```kotlin
fun findAllParking(): List<Parking> = parking.values.toList()
```